### PR TITLE
Python programiz in Python Language

### LINKED ISSUE #612: [Add] python course by programiz
### Site URL

https://www.programiz.com/python-programming

### Description of the above provided site

This is a course for learning Python for beginners. 

### Category

Languages

### Sub Category

Python

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [x] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by coder-chahal**
## Fixes Issue
Closes #612 

## Changes proposed

Add new resource in Python, Subcategory of "Languages".

## Screenshots

issue

### CODE REVIEW

**Comment by rupali-codes** [comment]
Hey, please update title of your PR (ex. chore: added python programmiz in Python Languages). Also, read contributing guidelines for more information :)

**Comment by coder-chahal** [comment]
@rupali-codes title updated.

File: database/languages/python.json [hunk 1]
```diff
@@ -5,5 +5,12 @@
         "url": "https://www.codecademy.com/learn/learn-python-3",
         "category": "languages",
         "subcategory": "python"
+    },
+    {
+        "name": "Python Programiz",
+        "description": "This Python course cover all the fundamental concepts of Python.",
```
Comment (CBID2) [inline database/languages/python.json:11]: [Proofreading] 
```suggestion
        "description": "This course covers all the fundamental concepts of Python.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @coder-chahal! 😄  You're off to a great start! :) Just fix the proofreading error that I mentioned in my inline comment and you're good to go! 😄 👍🏾

**Comment by coder-chahal** [comment]
thank you @CBID2!!

**Comment by coder-chahal** [comment]
hey @CBID2 proofreading error fixed. Please merge the PR.

**Review by CBID2 — APPROVED** [review]
Great job @coder-chahal! 😊 LGTM! 😊

**Review by rupali-codes — APPROVED** [review]
looks good

**Pull request merged**